docs/src/forc/plugins.md

```diff
@@ -0,0 +1,37 @@
+# Plugins
+
+The Fuel ecosystem has a few plugins which can be easily installed via cargo, and anyone can write their own plugins.
+
+Let's install a starter plugin, `forc-gm`, and take a look at how it works underneath:
+
+```sh
+cargo install forc-gm
```

Comment:
Re [this discussion](https://github.com/FuelLabs/sway/pull/1373#discussion_r857058919) - looks like we should publish if we want to install it like this :)

Reply:
i can also see why perhaps we might not want to publish this since it's just a sample for demonstration purposes, and if so we can remove this instruction

Reply:
I think it's nice to get the experience of what it's like to install a custom plugin too. Though I suppose users already get this through `forc-fmt` etc. That said, I'd be happy to see us publish this along with the rest of the crates even if it is only a demo - all our publishing is automated and the occasional version bump shouldn't be much of a hassle. I think it's probably important we at least own the name on crates.io either way.